scraper.rb

```diff
@@ -81,19 +86,28 @@ def constituencies
 # New layout
 # ----------
 
-new_format_terms = {
+new_terms = {
   '2016' => 'https://en.wikipedia.org/wiki/Template:MembersAlthing2016',
   '2013' => 'https://en.wikipedia.org/wiki/Template:MembersAlthing2013',
 }
 
-new_format_terms.each do |term, url|
-  page = MembersPageWithAreaTable.new(response: Scraped::Request.new(url: url).response)
-  members = page.members.each { |m| m[:term] = term }
-  # puts members
-  puts "#{term}: #{members.count}"
-  ScraperWiki.save_sqlite(%i(name term), members)
+old_revisions = {
+  '2013' => 'https://en.wikipedia.org/w/index.php?title=Template:MembersAlthing2013&direction=prev&oldid=714280236',
+}
+
+def scrape_new_format_terms(*term_hashes)
+  term_hashes.each do |term_hash|
+    term_hash.each do |term, url|
+      members = (scrape url => MembersPageWithAreaTable).members.each do |m|
+        ScraperWiki.save_sqlite(%i(name term), m.merge(term: term))
```

Comment:
This is all still quite confusing, where you're doing too much inside the `save` call, and with a slightly odd `members` variable that represents an intermediate stage, solely so you can output a count. I'd suggest doing this the same way as in the other scrapers, where you build up `data`, then save it.